sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/http/HttpDownloadFileTask.java

```diff
@@ -0,0 +1,174 @@
+package com.microsoft.appcenter.distribute.download.http;
+
+import android.annotation.SuppressLint;
+import android.net.TrafficStats;
+import android.os.AsyncTask;
+
+import com.microsoft.appcenter.distribute.ReleaseDetails;
+import com.microsoft.appcenter.http.TLS1_2SocketFactory;
+import com.microsoft.appcenter.utils.AppCenterLog;
+import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URL;
+import java.net.URLConnection;
+
+import javax.net.ssl.HttpsURLConnection;
+
+import static com.microsoft.appcenter.distribute.DistributeConstants.DOWNLOAD_FILES_PATH;
+import static com.microsoft.appcenter.distribute.download.ReleaseDownloader.Listener;
+import static com.microsoft.appcenter.distribute.download.http.HttpConnectionReleaseDownloader.PREFERENCE_KEY_DOWNLOADED_FILE;
+import static com.microsoft.appcenter.http.HttpUtils.CONNECT_TIMEOUT;
+import static com.microsoft.appcenter.http.HttpUtils.READ_TIMEOUT;
+import static com.microsoft.appcenter.http.HttpUtils.THREAD_STATS_TAG;
+import static com.microsoft.appcenter.http.HttpUtils.WRITE_BUFFER_SIZE;
+
+/**
+ * <h3>Description</h3>
+ * <p>
+ * Internal helper class. Downloads an .apk from AppCenter and stores
+ * it on external storage. If the download was successful, the file
+ * is then opened to trigger the installation.
+ **/
+@SuppressLint("StaticFieldLeak")
```

Comment:
Since there's just one static field, I think it should prefix field and not entire class

Reply:
Actually it's useless at all now

Reply:
```suggestion
```